core/server/apps/amp/lib/helpers/amp_components.js

```diff
@@ -0,0 +1,29 @@
+// # Amp Components Helper
+// Usage: `{{amp_components}}`
+//
+// Reads through the AMP HTML and adds neccessary scripts
+// for each extended component
+// If more components need to be added, we can add more scripts.
```

Comment:
👍 i've tested a non supported component (audio), and it still works, because `<audio>` html5 is not replaced by amp-audio. thats nice